testing/scenario_app/bin/android_integration_tests.dart

```diff
@@ -232,6 +258,8 @@ Future<void> _run({
         if (smokeTestFullPath != null)
           '-e class $smokeTestFullPath',
         'dev.flutter.scenarios.test/dev.flutter.TestRunner',
+        if (graphicsBackend != _GraphicsBackend.skia)
+          '-e enable-impeller',
```

Comment:
I think technically this will give you the autoselect option which may fall back to GLES today (or Skia in the future)

you should add: `-e impeller_backend vulkan` from https://github.com/flutter/engine/blob/main/common/settings.h#L225

Reply:
Oh, if it's that easy, let me just add OpenGLES too (@dnfield wanted to make sure it doesn't like, regress).